src/main.rs

```diff
@@ -1,3 +1,33 @@
-fn main() {
-    println!("Hello, world!");
+mod database;
+mod song_scores;
+
+
+use axum::Extension;
+use axum::routing::post;
+use axum::Router;
+use database::db::Database;
+use database::migrate::migrate_database;
+use song_scores::http::actions::new_submission::new_submission;
+use std::sync::Arc;
+
+#[tokio::main]
+async fn main() {
+    let _ = dotenv::dotenv();
+
+    let db = Arc::new(Database::new().await.session);
+
+    let _ = migrate_database(&db).await;
```

Comment:
Toda vez que startar a api é pra rodar todas as migrations mesmo?

Reply:
For now I'm using as my testing environment. Probably I can fetch the database metadata and validate if the schema still the same. 

But for now, I'll just let like that.